### PR TITLE
docs for firefox suggest data

### PR DESCRIPTION
**Pull request by rebecca-burwei**
@xluo-ds

### CODE REVIEW

File: src/datasets/other/suggest/suggest.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Search queries and Merino
+
+1. Search queries sent to Merino by Firefox are logged by Merino.
+  Full documentation of the probes is [here](https://firefox-source-docs.mozilla.org/browser/urlbar/firefox-suggest-telemetry.html#merino-search-queries). For more information about the much shorter retention periods, security and access controls on this data, see the [Search Terms Data Access Policy](https://docs.google.com/document/d/11rOM3r5AOPUrqDnCAODY7gknxnqtjphgINSK5oAR9T4/edit#) (Mozilla internal only).
+
+2. Merino responses as seen from Firefox.
+  We collect data about Merino's response times and response types in Firefox using the standard (legacy) Telemetry system as Histograms. Full documentation of the probes is [here](https://firefox-source-docs.mozilla.org/browser/urlbar/firefox-suggest-telemetry.html#histograms).
+
+3. Service and operational data on Merino
+  We also collect data about Merino as a service from Merino directly. Full documentation of the data is here.
```
Comment (rebecca-burwei) [inline src/datasets/other/suggest/suggest.md:42]: @Dexterp37 do you know where documentation on Merino data is?

**Review by skahmann3 — APPROVED** [review]
LGTM

File: src/datasets/other/suggest/suggest.md (hunk 1 above)
Comment (Dexterp37) [reply src/datasets/other/suggest/suggest.md:42]: @rebecca-burwei yes! The docs live [here](https://mozilla-services.github.io/merino/data.html)
```suggestion
  We also collect data about Merino as a service from Merino directly. Full documentation of the data is [here](https://mozilla-services.github.io/merino/data.html). 
```

**Comment by rebecca-burwei** [comment]
@xluo-ds Sydney approved, so I'm going to merge this in. Still very open to your feedback! Feel free to open an ticket if you have some.

**Comment by xluo-ds** [comment]
LGTM, thank you!

**Pull request merged**